charts/substra-backend/templates/deployment-backend.yaml

```diff
@@ -37,9 +117,9 @@ spec:
           {{- end }}
           command: ["/bin/bash"]
           {{- if eq .Values.backend.settings "prod" }}
-          args: ["-c", "python manage.py migrate && python3 manage.py collectstatic --noinput && uwsgi --http :8000 --module backend.wsgi --static-map /static=/usr/src/app/backend/statics --master --processes 4 --threads 2 --need-app --env DJANGO_SETTINGS_MODULE=backend.settings.server.{{ .Values.backend.settings }} "]
```

Comment:
Can you also move the collectstatic command in the init-container? I could help restart time of the pod (you might need to share a volume for the static files, you can use an emptyDir)